pages/movie.js

```diff
@@ -1,14 +1,22 @@
-import { Container, Image, Text, Title } from "@mantine/core";
+import { Button, Container, Image, Text, Title } from "@mantine/core";
 import Link from "next/link";
 import { useRouter } from "next/router";
+import { ArrowNarrowLeft } from "tabler-icons-react";
 
 const MoviePage = () => {
   const router = useRouter();
   const { id, image, title, year } = router.query;
 
   return (
     <Container>
-      <Link href="/">Back</Link>
```

Comment:
I guess we are gonna have the back button in other places,
What do you think about having it as a separate component and reuse it everywhere?

Reply:
Let's do that.